### DISCUSSION TITLE
Rancher Desktop installing OK but activating k8s fails

### DISCUSSION THREAD

**Discussion by lizfrench-esgglobal** [rancher-sandbox/rancher-desktop#7365] [Q&A]
Hi, I'm running Rancher Desktop V1.15.0 and k8s 1.27.3, these both work fine for all users but hitting an issue on one laptop where k8s won't install giving this error below.  However,  the linux distribution can see the internet, I have no internet connection issues.  I've tried re-installing on Ubuntu and the rancher desktop default distribution.  Can anyone shed any light on what this could possibly be.

```2024-08-20T12:43:32.456Z: Got error while checking connectivity: FetchError: request to
http://docs.rancherdesktop.io/
failed, reason: read ECONNRESET
2024-08-20T12:43:37.476Z: Running connectivity test with timeout of 5000 ms
2024-08-20T12:43:37.476Z: Connection test completed successfully
2024-08-20T13:04:49.361Z: Running check CONNECTED_TO_INTERNET
2024-08-20T13:04:49.427Z: Check CONNECTED_TO_INTERNET result: {"description":"The application cannot reach the general internet for updated kubernetes versions and other components, but can still operate.","passed":true,"fixes":[]}```

**Comment by jandubois** [comment]
Rancher Desktop fetches `http://docs.rancherdesktop.io` and verifies that the response is a 301 redirect and the `Location` header includes the `docs.rancherdesktop.io` substring to make sure you have a direct connection and are not trapped by a captive portal of a proxy server.

Can you verify that you get the expected data, e.g. with

```console
$ curl -I http://docs.rancherdesktop.io
HTTP/1.1 301 Moved Permanently
Connection: keep-alive
Content-Length: 162
Server: GitHub.com
Content-Type: text/html
x-origin-cache: HIT
Location: https://docs.rancherdesktop.io/
```
